README.md

```diff
@@ -123,6 +157,7 @@ git rebase master        # Rebase current branch on top of master branch
 git remote              # Show your current remotes
 git remote -v           # Show your current remotes and their URLs
 git push                # Publish your commits to the upstream master of your currently checked out branch
+git push -u origin my-branch            # To push newly created branch to remote repo.
```

Comment:
Should maybe mention why we add the `-u` ?

Reply:
clarified that part too.